source/confirm_references.html.md.erb

```diff
@@ -0,0 +1,35 @@
+---
+title: Confirm references
+weight: 50
+---
+
+# Confirm references submitted
+The references of the candidate have been submitted.
+
+```
+POST /applications/:uuid/confirm_references
+```
+
+## Example request
+
+```
+POST /applications/aa546741-b54a-403c-a67a-7f5aec1939d1/confirm_references
+```
+
+Example request body:
+
+```json
+<%= {} %>
```

Comment:
We've specified that a `date` is an attribute, shouldn't the JSON request include this? Same question for the other endpoints.

Reply:
Not sure if need a date 🤔 
@duncanjbrown do you think we should use a `date` as an attribute on the action `confirm_references` (and confirm_condition_met, and confirm_onboarding)?

Reply:
During a chat with Duncan,we decided that at the moment we can assume that the date is equal to 'current date'. Also, there is not yet evidence that the field date is necessary.